src/api/app/models/workflow/step/branch_package_step.rb

```diff
@@ -48,6 +48,10 @@ def target_project
     Project.find_by(name: target_project_name)
   end
 
+  def target_project_url
```

Comment:
This method makes not much sense and just hides what this really is below...

Reply:
DRY is not about typing :) The ` WorkflowRun.event_source_url` is perfectly DRY...